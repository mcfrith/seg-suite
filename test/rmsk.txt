585	463	13	6	17	chr1	10000	10468	248945954	+	(TAACCC)n	Simple_repeat	Simple_repeat	1	463	0	1
585	558	111	59	30	chr1	10468	11463	248944959	-	TAR1	Satellite	telo	1	1023	399	2
585	64	242	178	25	chr1	11502	11676	248944746	-	L1MC	LINE	L1	5448	5647	2235	3
585	35	291	19	9	chr1	11676	11780	248944642	-	MER5B	DNA	hAT-Charlie	1	105	73	4
585	30	247	35	0	chr1	15264	15349	248941073	-	MIR1_Amn	SINE	MIR	62	149	81	5
585	18	232	0	20	chr1	15797	15849	248940573	+	(TGCTCC)n	Simple_repeat	Simple_repeat	1	51	0	6
585	34	296	255	0	chr1	16361	16459	248939963	-	Charlie15a	DNA	hAT-Charlie	2	124	100	7
585	18	137	0	0	chr1	16712	16744	248939678	+	(TGG)n	Simple_repeat	Simple_repeat	1	32	0	8
585	21	315	217	0	chr1	18956	19048	248937374	+	L2a	LINE	L2	2993	3104	322	9
585	233	323	118	17	chr1	19971	20727	248935695	+	L3	LINE	CR1	2680	3510	589	10
585	117	324	159	27	chr1	21949	22345	248934077	+	MLT1K	LTR	ERVL-MaLR	16	462	133	11
585	113	307	42	0	chr1	23120	23361	248933061	-	MIR	SINE	MIR	7	257	5	12
585	51	296	236	9	chr1	23836	24010	248932412	+	L2b	LINE	L2	2967	3179	196	13
585	56	295	282	4	chr1	24072	24249	248932173	+	MIR	SINE	MIR	34	259	3	14
585	36	338	91	46	chr1	24261	24447	248931975	+	L2b	LINE	L2	3231	3424	2	13
585	10	302	59	125	chr1	25891	25959	248930463	+	MIR	SINE	MIR	84	147	115	15
585	18	245	42	138	chr1	26356	26427	248929995	+	MIR1_Amn	SINE	MIR	109	173	57	16
585	47	358	242	19	chr1	26573	26790	248929632	-	L2c	LINE	L2	3116	3382	5	17
585	291	88	4	0	chr1	26790	27053	248929369	+	AluSp	SINE	Alu	1	264	49	18
585	47	358	242	19	chr1	27053	27220	248929202	-	L2c	LINE	L2	2915	3115	272	17
585	189	155	148	0	chr1	27268	27540	248928882	+	MER33	DNA	hAT-Charlie	1	299	25	19
585	24	352	126	28	chr1	27790	27986	248928436	-	MIRc	SINE	MIR	32	228	2	20
585	33	336	33	26	chr1	28150	28300	248928122	-	MIRb	SINE	MIR	108	258	10	21
585	12	157	42	64	chr1	29744	29792	248926630	+	(ACGCCC)n	Simple_repeat	Simple_repeat	1	47	0	22
585	200	222	27	10	chr1	29900	30196	248926226	+	L1MB3	LINE	L1	5882	6182	1	23
585	129	155	32	16	chr1	30342	30532	248925890	-	MER53	DNA	hAT	1	193	0	24
585	140	291	105	263	chr1	30694	31133	248925289	+	MLT1A1	LTR	ERVL-MaLR	2	385	23	25
585	50	297	281	4	chr1	31258	31435	248924987	+	MIRc	SINE	MIR	5	231	37	26
585	303	154	0	0	chr1	31435	31733	248924689	+	AluJb	SINE	Alu	1	298	14	27
585	50	297	281	4	chr1	31734	31752	248924670	+	MIRc	SINE	MIR	232	253	15	26
585	79	251	119	27	chr1	32840	33041	248923381	+	MIR	SINE	MIR	2	220	42	28
585	318	183	49	0	chr1	33046	33456	248922966	+	L1MB5	LINE	L1	5745	6174	0	29
585	16	250	0	0	chr1	33468	33508	248922914	+	FLAM_C	SINE	Alu	4	43	100	30
585	584	94	2	22	chr1	33530	34097	248922325	-	L1PA6	LINE	L1	5596	6151	3	31
585	47	396	4	108	chr1	34114	34349	248922073	-	MLT1J	LTR	ERVL-MaLR	116	328	184	32
585	35	333	7	88	chr1	34414	34562	248921860	-	L2c	LINE	L2	3047	3183	204	33
585	168	286	134	16	chr1	34564	34958	248921464	-	MLT1J2	LTR	ERVL-MaLR	28	448	0	34
585	38	324	119	18	chr1	35215	35366	248921056	-	MIRc	SINE	MIR	91	256	12	35
585	134	135	0	0	chr1	35366	35499	248920923	+	AluJr	SINE	Alu	1	133	179	36
585	259	191	36	28	chr1	37044	37431	248918991	+	Charlie5	DNA	hAT-Charlie	2234	2623	1	37
585	21	345	8	104	chr1	37734	37860	248918562	+	L2c	LINE	L2	3272	3386	1	38
585	66	209	164	5	chr1	38070	38229	248918193	+	L2a	LINE	L2	2717	2900	519	39
585	706	242	140	12	chr1	38238	39462	248916960	+	MLT1F2-int	LTR	ERVL-MaLR	109	1487	2	40
585	198	293	109	75	chr1	39464	39623	248916799	+	MLT1F2	LTR	ERVL-MaLR	1	174	386	40
585	322	120	0	0	chr1	39623	39923	248916499	+	AluSx	SINE	Alu	1	300	12	41
585	198	293	109	75	chr1	39925	40297	248916125	+	MLT1F2	LTR	ERVL-MaLR	175	548	12	40
585	108	308	32	32	chr1	40339	40624	248915798	+	L2a	LINE	L2	2931	3215	211	39
585	66	130	43	0	chr1	40639	40731	248915691	-	AluSg7	SINE	Alu	203	298	11	42
585	98	400	96	6	chr1	40738	41072	248915350	-	LTR16C	LTR	ERVL	10	373	116	43
585	366	361	151	50	chr1	41160	42284	248914138	-	ERV3-16A3_I-int	LTR	ERVL	3882	5114	112	44
