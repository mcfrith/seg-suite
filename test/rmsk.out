   SW  perc perc perc  query      position in query           matching       repeat              position in  repeat
score  div. del. ins.  sequence    begin     end    (left)    repeat         class/family         begin  end (left)   ID

  463   1.3  0.6  1.7  chr1        10001   10468 (248945954) +  (TAACCC)n      Simple_repeat            1  463    (0)      1
  559  11.1  5.9  3.0  chr1        10469   11463 (248944959) C  TAR1           Satellite/telo       (399) 1023      1      2
   64  24.6 17.7  2.5  chr1        11502   11676 (248944746) C  L1MC           LINE/L1             (2234) 5648   5448      3
   35  29.1  1.9  0.9  chr1        11677   11780 (248944642) C  MER5B          DNA/hAT-Charlie       (73)  105      1      4
   30  25.8  4.5  0.0  chr1        15265   15353 (248941069) C  MIR1_Amn       SINE/MIR              (81)  149     57      5
   18  23.2  0.0  2.0  chr1        15798   15849 (248940573) +  (TGCTCC)n      Simple_repeat            1   51    (0)      6
   34  29.6 25.5  0.0  chr1        16362   16459 (248939963) C  Charlie15a     DNA/hAT-Charlie      (100)  124      2      7
   18  13.7  0.0  0.0  chr1        16713   16744 (248939678) +  (TGG)n         Simple_repeat            1   32    (0)      8
   11  35.5  8.6 15.1  chr1        18418   18649 (248937773) +  L1M2b          LINE/L1                744  962 (6247)      9
   21  32.5 19.3  0.4  chr1        18908   19049 (248937373) +  L2a            LINE/L2               2941 3104  (322)     10
  232  32.3 11.8  1.7  chr1        19972   20727 (248935695) +  L3             LINE/CR1              2680 3510  (589)     11
  116  32.3 15.9  2.7  chr1        21950   22344 (248934078) +  MLT1K          LTR/ERVL-MaLR           16  461  (134)     12
  113  30.7  4.2  0.0  chr1        23121   23361 (248933061) C  MIR            SINE/MIR               (5)  257      7     13
   50  29.6 23.6  0.9  chr1        23837   24010 (248932412) +  L2b            LINE/L2               2967 3179  (196)     14
   55  29.5 28.2  0.4  chr1        24073   24249 (248932173) +  MIR            SINE/MIR                34  259    (3)     15
   36  33.6  9.1  4.6  chr1        24261   24447 (248931975) +  L2b            LINE/L2               3230 3424    (2)     14
   14  36.2 22.3  0.8  chr1        25182   25284 (248931138) C  MIRb           SINE/MIR              (21)  247    123     16
   10  30.2  5.9 12.5  chr1        25892   25959 (248930463) +  MIR            SINE/MIR                84  147  (115)     17
   18  25.7  4.2 13.6  chr1        26357   26428 (248929994) +  MIR1_Amn       SINE/MIR               109  174   (56)     18
   47  35.8 24.2  1.9  chr1        26574   26790 (248929632) C  L2c            LINE/L2                (5) 3382   3116     19
  292   8.8  0.4  0.0  chr1        26791   27053 (248929369) +  AluSp          SINE/Alu                 1  264   (49)     20
   47  35.8 24.2  1.9  chr1        27054   27220 (248929202) C  L2c            LINE/L2              (272) 3115   2915     19
  189  15.6 14.6  0.0  chr1        27267   27541 (248928881) +  MER33          DNA/hAT-Charlie          1  300   (24)     21
   23  35.2 12.5  2.8  chr1        27790   27986 (248928436) C  MIRc           SINE/MIR               (1)  229     32     22
   33  33.6  3.3  2.6  chr1        28151   28300 (248928122) C  MIRb           SINE/MIR              (10)  258    108     23
   12  15.7  4.2  6.4  chr1        29745   29792 (248926630) +  (ACGCCC)n      Simple_repeat            1   47    (0)     24
  199  22.2  2.7  1.0  chr1        29901   30196 (248926226) +  L1MB3          LINE/L1               5882 6182    (1)     25
  129  15.5  3.2  1.6  chr1        30343   30532 (248925890) C  MER53          DNA/hAT                (0)  193      1     26
  140  29.1 10.5 26.3  chr1        30695   31133 (248925289) +  MLT1A1         LTR/ERVL-MaLR            2  385   (23)     27
   50  29.6 27.9  0.4  chr1        31259   31435 (248924987) +  MIRc           SINE/MIR                 5  231   (37)     28
  303  15.4  0.0  0.0  chr1        31436   31733 (248924689) +  AluJb          SINE/Alu                 1  298   (14)     29
   50  29.6 27.9  0.4  chr1        31735   31753 (248924669) +  MIRc           SINE/MIR               232  254   (14)     28
   79  25.1 11.9  2.7  chr1        32841   33041 (248923381) +  MIR            SINE/MIR                 2  220   (42)     30
  317  18.3  4.9  0.0  chr1        33047   33456 (248922966) +  L1MB5          LINE/L1               5745 6174    (0)     31
   19  29.2  6.2  0.0  chr1        33467   33514 (248922908) +  Alu            SINE/Alu                 2   52  (260)     32
  584   9.4  0.2  2.2  chr1        33531   34097 (248922325) C  L1PA6          LINE/L1                (3) 6151   5596     33
   47  39.6  0.4 10.8  chr1        34115   34349 (248922073) C  MLT1J          LTR/ERVL-MaLR        (184)  328    116     34
   34  33.3  0.7  8.8  chr1        34415   34562 (248921860) C  L2c            LINE/L2              (204) 3183   3047     35
  167  28.4 14.1  0.2  chr1        34565   34934 (248921488) C  MLT1J2         LTR/ERVL-MaLR          (0)  448     28     36
   38  32.4 11.9  1.8  chr1        35216   35366 (248921056) C  MIRc           SINE/MIR              (12)  256     91     37
  136  13.5  0.0  0.0  chr1        35367   35499 (248920923) +  AluJr          SINE/Alu                 1  133  (179)     38
  260  19.1  3.6  2.8  chr1        37045   37431 (248918991) +  Charlie5       DNA/hAT-Charlie       2234 2623    (1)     39
   21  34.5  0.8 10.4  chr1        37735   37860 (248918562) +  L2c            LINE/L2               3272 3386    (1)     40
   65  21.1 16.1  0.5  chr1        38068   38229 (248918193) +  L2a            LINE/L2               2714 2900  (519)     41
  706  24.2 14.0  1.2  chr1        38239   39462 (248916960) +  MLT1F2-int     LTR/ERVL-MaLR          109 1487    (2)     42
  198  29.1 10.9  7.5  chr1        39465   39623 (248916799) +  MLT1F2         LTR/ERVL-MaLR            1  174  (386)     42
  323  12.0  0.0  0.0  chr1        39624   39923 (248916499) +  AluSx          SINE/Alu                 1  300   (12)     43
